pages/communities/modules/[module]/index.tsx

```diff
@@ -1,7 +1,74 @@
+import {Thread} from 'emse-ui'
+import {Input} from 'emse-ui'
+
 const Module = () => {
+    const userName="joeldesante"
+    const userEmail="jdsante@odu.edu"
+    const moduleName ="This is the name of the module"
     return (
-        <div className='flex justify-center items-center h-screen'>
-            <p>Communities Module</p>
+        <div className='m-8 flex'>
+            <div className='m-10 grow'>
+                <p className="text-2xl font-bold">{moduleName}</p>
+                <div className="flex my-2">
+                <img
+					src="https://i.pinimg.com/custom_covers/222x/85498161615209203_1636332751.jpg"
+					alt="profile image"
+					className="shadow-lg rounded-full max-w-full h-4 align-middle border-none"
+				/>
+                    <small className="px-2">{userName}</small>
+                     <small className="px-3">
+                        <ul className="list-disc">
```

Comment:
This should be a `&bull;` character. Not a list element
https://www.w3schools.com/charsets/ref_html_entities_4.asp